File: algo-c-to-_/examples/dayweek_.py
#
#	from src/dayweek.c
#
#	a part of main	to	dayweek, initArray
#

from dayweek import dayweek, initArray

a = initArray()

for i in range(21, 32):
    print("{0:4d}/{1:02d}/{2:02d} {3}".format(2019, 12, i, a[dayweek(2019,12,i)]))

for i in range(1, 12):
    print("{0:4d}/{1:02d}/{2:02d} {3}".format(2020, 1, i, a[dayweek(2020,1,i)]))

File: porting-from-algo-c/examples/dayweek.lua
--
--	from src/dayweek.c
--
--	a part of main	to	dayweek, initArray
--

local M = require 'dayweek'

local dayweek, initArray = M.dayweek, M.initArray

do
	local a = initArray()

	for i=21,31 do
		print(("%4d/%02d/%02d %s"):format(2019, 12, i, a[dayweek(2019,12,i)]))
	end

	for i=1,11 do
		print(("%4d/%02d/%02d %s"):format(2020, 1, i, a[dayweek(2020,1,i)]))
	end
end
